.gitignore

```diff
@@ -10,7 +10,7 @@ __pycache__/
 local_settings.py
 db.sqlite3
 db.sqlite3-journal
-media
+media/profile_pictures/
```

Comment:
I think that this isn't needed anymore. the old line is ok.

Reply:
I think it's still a good practice to explicitly mention the images we don't want to ignore in this file. So, if anyone wants to make the rest of the changes, please handle this one as well (as you deem appropriate, I'm ok either way).

Reply:
Is this resolved??? 
If so please click the `Resolve conversation` button below.

Reply:
Changed to `media`.